pulsar-metadata/src/test/java/org/apache/pulsar/metadata/BacklogQuotaCompatibilityTest.java

```diff
@@ -26,7 +26,7 @@
 
 import java.io.IOException;
 
-import static org.junit.Assert.assertEquals;
+import static org.testng.Assert.assertEquals;
```

Comment:
Please take care of assertEquals order

Reply:
Already swapped

Reply:
This was originally wrong, but it was correct after using testng

Reply:
@hangc0276 PTAL